config/helm/aws-node-termination-handler/templates/clusterrole.yaml

```diff
@@ -37,3 +37,16 @@ rules:
     - daemonsets
   verbs:
     - get
+{{- if .Values.emitKubernetesEvents }}
+- apiGroups:
+    - ""
+  resources:
+    - events
+  verbs:
+    - create
+    - get
+    - list
+    - patch
+    - update
+    - watch
```

Comment:
are all these necessary?

Reply:
Probably not. Removed them all but `create`.